docs/en/Debugging-Guide.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: global
+title: Debugging Guide
+group: Resources
+---
+
+* Table of Contents
+{:toc}
+
+This page is a collection of high-level guides and tips regarding how to diagnose issues encountered in
+Alluxio.
+
+Note: this doc is not intended to be the full list of Alluxio questions.
+Feel free to post questions on [Alluxio Mailing List](https://groups.google.com/forum/#!forum/alluxio-users).
+
+## Environment
+
+Alluxio can be configured under a variety of modes, in different production environments.
+Please make sure the Alluxio version being deployed, is update-to-date and supported.
+
+When posting questions on [Mailing List](https://groups.google.com/forum/#!forum/alluxio-users),
+please attach the full environment information, including
+- Alluxio version
+- OS version
+- Java version
+- UnderFileSystem type and version
+- Computing framework type and version
+- Cluster information
```

Comment:
you might want to add some more detail to Cluster information about what sort of information you're looking for